proof/refine/ARM/KHeap_R.thy

```diff
@@ -1612,6 +1612,14 @@ lemma get_reply_corres:
   apply (simp add: other_obj_relation_def)
   done
 
+lemma get_reply_tcb_corres:
```

Comment:
This is one of the cases where we have an abstract function but no concrete one. @VictorPhan what convention are we going with for cases like this? I guess the alternative name would be `getReplyTCB_corres`

Reply:
My immediate instinct was to go for something like `getReply_TCB_corres` since we're calling `getReply` at the core and a `TCB` is returned from the body. The naming could be confused with passing in a `TCB` to`getReply` though, so another possibility is the more explicit `replyTCB_getReply_corres`.

Reply:
Out of those two I prefer `getReply_TCB_corres`. If we were going more explicit I would prefer `getReply_replyTCB_corres`, but that seems a bit repetitive.

Reply:
I'm happy with `getReply_TCB_corres`, though I'm also okay with having a bit of repetition for explicitness.

Reply:
I think I’d also slightly prefer the shorter `getReply_TCB_corres`.